python/xorbits/_mars/dataframe/datasource/read_parquet.py

```diff
@@ -493,12 +493,32 @@ def _tile_no_partitioned(cls, op: "DataFrameReadParquet"):
     def _tile_http_url(cls, op: "DataFrameReadParquet"):
         out_chunks = []
         out_df = op.outputs[0]
-        for i, url in enumerate(op.path):
+        z = None
+        zip_filename = None
+        if op.path[0].endswith(".zip"):
+            import urllib.request
+
+            zip_filename, _ = urllib.request.urlretrieve(op.path[0])
+            z = zipfile.ZipFile(zip_filename)
```

Comment:
If the ZipFile supports a file-like object, we can use a file stream opened by fsspec instead of download.

@aresnow1 How about leave the code as it is, then we can solve this problem better by refectoring the filesystem to fsspec.